Makefile

```diff
@@ -262,22 +267,22 @@ else
 	@npm run test -- --local firefox
 endif
 
-test-py: dist/urth
+test-py: dist/urth dist/declarativewidgets
 	@echo 'Running python tests in $(PYTHON)...'
 	@$(MAKE) _test-py-$(PYTHON)
 
-_test-py-python2: EXTENSION_DIR=/opt/conda/envs/python2/lib/python2.7/site-packages/urth
-_test-py-python2: CMD=python --version; python -m unittest discover $(EXTENSION_DIR) "test*[!_py3].py"
+_test-py-python2: EXTENSION_DIR=/opt/conda/envs/python2/lib/python2.7/site-packages
+_test-py-python2: CMD=python --version; python -m unittest discover $(EXTENSION_DIR)/declarativewidgets "test*[!_py3].py"
 _test-py-python2: PYTHON_SETUP_CMD=source activate python2; pip install -U mock $(PIP_OPTS);
 _test-py-python2: _test-py
 
-_test-py-python3: EXTENSION_DIR=/usr/local/lib/python3.4/dist-packages/urth
+_test-py-python3: EXTENSION_DIR=/usr/local/lib/python3.4/dist-packages
 _test-py-python3: CMD=python --version; python -m unittest discover $(EXTENSION_DIR)
```

Comment:
Is the same change that was made for `CMD` on `_test-py-python2` not needed for `_test-py-python3`?